draft-ietf-httpbis-safe-method-w-body.xml

```diff
@@ -791,7 +804,7 @@ See stored query at "/contacts/stored-queries/42".
 </artwork>
         <t>
           This is similar to including Location on a direct response, except that no result
-          for the query is returned. This allows the server to only generate an alternative
+          for the query is returned. This allows the server to only generate or reuse an alternative
           resource. This resource could then be used as shown in
           <xref target="example.location"/>.
```

Comment:
Do you want to observe that the server might be optimizing for multiple subsequent GET requests to the identified resource.

Reply:
But that would be the same for 2xx and "Location", no?